src/index.js

```diff
@@ -207,6 +207,7 @@ class Libp2p extends EventEmitter {
     }
 
     // Create keychain
+      // TODO: Decide how to change these options when the keychain password is being changed
```

Comment:
Do you mean the `const keychainOpts = Keychain.generateOptions()`? I think we can just run it again to get the options. What do you think?

Reply:
Yes that sounds good to me.